repos.txt

```diff
@@ -47,6 +50,7 @@ java-dialogflow
 java-dialogflow-cx
 java-dlp
 java-dms
+java-dns
```

Comment:
While this is a handwritten library, as long as there're no dedeicated team to maintain the repositories, we maintain it. We should add it to monorepo for the sake of repository maintenance.